sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -536,6 +541,14 @@ void setSendingRequestHasStarted(boolean hasSendingRequestStarted) {
         this.sendingRequestHasStarted = hasSendingRequestStarted;
     }
 
+    void setFaultInjectionRuleId(String faultInjectionRUleId) {
+        this.faultInjectionRuleId = faultInjectionRUleId;
```

Comment:
faultInjectionRuleId should also be added to toString() implementation?

Reply:
good catch, added